src/CppUTest/CommandLineArguments.cpp

```diff
@@ -149,6 +155,21 @@ void CommandLineArguments::SetRepeatCount(int ac, const char *const *av, int& i)
 
 }
 
+void CommandLineArguments::SetShuffle(int ac, const char * const *av, int& i)
+{
+    shuffle_ = 1;
```

Comment:
Is this line needed?

Reply:
Yes - it's the code path that enables shuffling when command line switch includes plain "-s" (unit test: CommandLineArguments :: shuffleEnabled )